plugins/modules/net_tools/ldap/ldap_attr.py

```diff
@@ -35,6 +35,9 @@
     rules. This should work out in most cases, but it is theoretically
     possible to see spurious changes when target and actual values are
     semantically identical but lexically distinct.
+  - "The C(params) parameter was removed due to circumventing Ansible's parameter
+     handling.  The C(params) parameter started disallowing setting the bind_pw parameter in
```

Comment:
```suggestion
     handling.  The I(params) parameter started disallowing setting the I(bind_pw) parameter in
```

Reply:
Do you think `bind_pw` should use C() since it's part of an option value for `params`?

Reply:
It's an option key, no idea if there's a standard for these (probably `C(...)`?). But on the other hand, it is a module option (and its a key for `params` precisely because it is a module option), so I'd use `I(...)` in this case.